src/CodeStyle/CSharp/Analyzers/CSharpCodeStyleResources.resx

```diff
@@ -126,4 +126,48 @@
   <data name="Wrapping_preferences" xml:space="preserve">
     <value>Wrapping preferences</value>
   </data>
+  <data name="Make_readonly_fields_writable" xml:space="preserve">
```

Comment:
I am going to create a separate PR that reverts all the resx and xlf file changes for anyone who prefers to just look at the source code files.